Source/ui_android/java/com/ikimuhendis/ldrawer/ActionBarDrawerToggle.java

```diff
@@ -0,0 +1,182 @@
+package com.ikimuhendis.ldrawer;
+
+
+import android.content.res.Configuration;
+import android.graphics.drawable.Drawable;
+import android.os.Build;
+import android.support.v4.view.GravityCompat;
+import android.support.v4.widget.DrawerLayout;
+import android.support.v7.app.ActionBar;
+import android.support.v7.app.ActionBarActivity;
+import android.util.Log;
+import android.view.MenuItem;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+
+import com.virtualapplications.play.R;
```

Comment:
Humm, from what I understand, Ldrawer is an external library. Do we need to import stuff from the emulator in there?